### DISCUSSION TITLE
chromedriver 102

### DISCUSSION THREAD

**Discussion by lplazaromansymphony** [yarnpkg/berry#4511] [General]
hi, could we have please chromedriver 102? I can see here that is not updated yet https://yarnpkg.com/package/chromedriver thx!

**Comment by arcanis** [comment]
Because this version doesn't exist

**Reply by lplazaromansymphony** [reply]
not sure what you mean 
<<Chrome is up to date
Version 102.0.5005.61>>

**Reply by arcanis** [reply]
Then bring that to the ChromeDriver team - this is a completely different project, we have no control over what they publish.
